library/src/main/java/com/shotcutter/library/converter/ConverterService.java

```diff
@@ -29,9 +29,10 @@
         log.info("Converters map successfully initialized");
     }
 
-    public <Original, Target> Optional<Target> convertTo(Original item, Class<Target> targetClass) {
+    public <Original, Target> Target convertTo(Original item, Class<Target> targetClass) {
         return getConverter(item.getClass(), targetClass)
-                .map(converter -> ((Converter<Original, Target>)converter).convert(item));
+                .map(converter -> ((Converter<Original, Target>)converter).convert(item))
+                .get();
```

Comment:
before get you can validate the chain, e.g. you can use elseThrow() if you don't have converter/object